cli/command/run/cmd.go

```diff
@@ -222,3 +230,18 @@ func enableCIProviders() {
 	gitlab.Enable()
 	jenkins.Enable()
 }
+
+func filterSuite(c *config.Project) error {
+	found := false
+	for _, s := range c.Suites {
+		if s.Name == suiteName {
+			found = true
+			c.Suites = []config.Suite{s}
+			break
```

Comment:
Not necessary to break the loop. You can just return nil here.

Reply:
oh sure.